terminus-ui/selection-list/src/selection-list.component.ts

```diff
@@ -927,4 +939,22 @@ export class TsSelectionListComponent implements
     this.inputElement.nativeElement.value = '';
   }
 
+  /**
+   * Scroll the active option into view
+   */
+  private scrollActiveOptionIntoView(): void {
+    const ctrlValue = this.selectionListFormControl.value || [];
```

Comment:
Is it possible that the control value is not an array? I know we default to an array during initialization..